server/portal/apps/workspace/api/views.py

```diff
@@ -145,30 +145,28 @@ def delete(self, request, *args, **kwargs):
         )
 
     def post(self, request, *args, **kwargs):
-        agave = request.user.tapis_oauth.client
+        tapis = request.user.tapis_oauth.client
         job_post = json.loads(request.body)
-        job_id = job_post.get('job_id')
+        job_uuid = job_post.get('job_uuid')
         job_action = job_post.get('action')
 
-        if job_id and job_action:
+        if job_uuid and job_action:
             # resubmit job
             if job_action == 'resubmit':
-                METRICS.info("user:{} is resubmitting job id:{}".format(request.user.username, job_id))
+                METRICS.info("user:{} is resubmitting job id:{}".format(request.user.username, job_uuid))
+                data = tapis.jobs.resubmitJob(jobUuid=job_uuid)
             # cancel job / stop job
             else:
-                METRICS.info("user:{} is canceling/stopping job id:{}".format(request.user.username, job_id))
-
-            data = agave.jobs.manage(jobId=job_id, body={"action": job_action})
+                METRICS.info("user:{} is canceling/stopping job id:{}".format(request.user.username, job_uuid))
+                data = tapis.jobs.cancelJob(jobUuid=job_uuid)
```

Comment:
Maybe we should either move this to a `delete` request, or handle an explicit `"cancel"` `job_action` case

Reply:
The `hideJob` replaces `deleteJob` in v3 (more accurate naming), which is under the `delete` request for the `JobsView`.
I'll went with the latter suggestion!